src/com/palmergames/bukkit/towny/TownyMessaging.java

```diff
@@ -47,22 +49,18 @@ public static void sendErrorMsg(String msg) {
 	 * @param msg the message to send
 	 */
 	public static void sendErrorMsg(Object sender, String msg) {
-		boolean isPlayer = false;
 		if (sender instanceof Player) {
-			isPlayer = true;
+			((Player) sender).sendMessage(PREFIX + ChatColor.RED + msg);
 		}
 
+		if (sender instanceof ConsoleCommandSender) {
+			((ConsoleCommandSender) sender).sendMessage(PREFIX + ChatColor.stripColor(msg));
+		}
+		
 		if (sender == null) {

```

Comment:
If the above if-statement passes, then this if-statement will always be false. Thus you can use an else-if here or you could move this if-statement above the previous one and use an else-if on the previous one.